torch/csrc/utils/schema_info.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <ATen/core/function_schema.h>
+#include <torch/csrc/jit/runtime/operator.h>
+
+namespace torch {
+namespace utils {
+
+/**
+ * class SchemaInfo
+ *
+ * Subclass of FunctionSchema that publicizes argument value specific operator
+ * behavior (mutation, aliasing, special cases, etc...)
+ */
+
+struct TORCH_API SchemaInfo : c10::FunctionSchema {
+ public:
+  explicit SchemaInfo(c10::FunctionSchema schema) : FunctionSchema(schema) {}
+  explicit SchemaInfo(const char* signature)
+      : FunctionSchema(torch::jit::getOperatorForLiteral(signature)->schema()) {
+  }
+};
```

Comment:
@davidberard98 Should we be concerned about performance impacts from virtualizing FunctionSchema? FunctionSchema is used in Core.

Reply:
Are any methods on `FunctionSchema` actually getting virtualized in this PR? It looks like `SchemaInfo inherits from `FunctionSchema` (but without virtualizing any methods). I think that should be fine.

Reply:
@bdhirsh Look at https://github.com/pytorch/pytorch/pull/80972/files for functions in `SchemaInfo` that we would be virtualizing. (They are currently not marked as virtual, but I expect that we would need them to be `virtual` in order for things to work as intended).

Reply:
Ah ok, in agreement with John then on composition over inheritance. `FunctionSchema` is probably hot-path enough that we want to avoid virtualizing some of its common methods.